src/layouts/default.vue

```diff
@@ -5,13 +5,24 @@
                 <v-app-bar-title class="ml-2">{{ appConfig?.title }}</v-app-bar-title>
             </router-link>
             <v-spacer></v-spacer>
-            <router-link to="/about">
-                <v-btn aria-label="About" icon>
-                    <v-icon>mdi-information-outline</v-icon>
-                    <v-tooltip activator="parent">About</v-tooltip>
-                </v-btn>
-            </router-link>
+            <v-app-bar-nav-icon aria-label="navigation menu" @click.stop="drawer = !drawer"></v-app-bar-nav-icon>
         </v-app-bar>
+        <v-navigation-drawer v-model="drawer" location="right" temporary app>
+            <v-list>
+                <v-list-item link to="/" prepend-icon="mdi-map-outline">
+                    <v-list-item-title>Map</v-list-item-title>
+                </v-list-item>
```

Comment:
Ideally this would default to show as the active item when first browse to site, but doesn't currently.